### ISSUE TITLE
LAN with IPv4 + IPv6

### ISSUE THREAD

**Issue by nhathaway** [john-tornblom/TVHGuide#34]
There is a problem on LANs with IPv6, where DNS provides both IPv4 and IPv6 addresses. TVHeadend itself only serves on IPv4, even if the box it is on is dual stack. When getting the name (from DNS), TVHGuide picks up the IPv6 address in preference over the IPv4, and tries to connect on IPv6, which results in connection refused.

**Comment by kecksk0enig** [comment]
I've  seen this too, solved it by using ip4 address on tvhguide. Did'nt know tvheadend was only serving @ ipv4.
Found 
 hint @ /etc/default/tvheadend 
#TVH_IPV6
#if set to 1 will enable IPv6 support
TVH_IPV6=0
I did set it to 1 restarted tvheadend and it works over ipv6 AND v4

**Comment by nhathaway** [comment]
Yes, or there's the "-6" command line option (I use a TBS MOI device which is somewhat non-standard).

Nevertheless, maybe the settings in the app needs a checkbox "IPv4 only", as this is the default for TVHeadend.
